### PR TITLE
Transferring the remaining windows to JS 

### PR DESCRIPTION
**Pull request by AyIong**
## What Does This PR Do

- Moved remaining windows sizes into JS
- Added titles for disposal bin, looks better
- Removed the text display of remaining time in nuke, now it's a button as if inserting a disc, only not active naturally
- Corrected the size of many windows
- Vendors, Air larms, Brig Timer, LawManager, radios and volume mixer, change the size of windows depending on their state, user or content. Unfortunately not in real time, you have to refresh the window
- Also the size of some windows can now be changed, in any case in the next versions of TGUI, this cannot be changed, as far as I know.

### CODE REVIEW

File: tgui/packages/tgui/interfaces/NuclearBomb.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const NuclearBomb = (props, context) => {
 
   if (!data.extended) {
     return (
-      <Window width={450} height={300}>
```
Comment (S34NW) [inline tgui/packages/tgui/interfaces/NuclearBomb.js:10]: Don't we still need this? Or are we just going with the default size

**Review by S34NW — APPROVED** [review]
Looks fine apart from that one change

File: tgui/packages/tgui/interfaces/NuclearBomb.js (hunk 1 above)
Comment (AyIong) [reply tgui/packages/tgui/interfaces/NuclearBomb.js:10]: Having poked around the window, didn't see any change in window size
Maybe it's best to delete it altogether

Comment (AyIong) [reply tgui/packages/tgui/interfaces/NuclearBomb.js:10]: I'd better double-check

Comment (AyIong) [reply tgui/packages/tgui/interfaces/NuclearBomb.js:10]: Okay, this is for the undeployed nuke, changed

**Review by Burzah — APPROVED** [review]
I am good with this. I will make sure the parent PR is updated and that your changes and name are added to it. Well done!

**Pull request merged**